poly/src/domain/radix2/fft.rs

```diff
@@ -61,12 +62,68 @@ impl<F: FftField> Radix2EvaluationDomain<F> {
 
     #[cfg(not(feature = "parallel"))]
     fn roots_of_unity(&self, root: F) -> Vec<F> {
-        crate::domain::utils::compute_powers_serial(self.size as usize, root)
+        compute_powers_serial(self.size as usize, root)
     }
 
     #[cfg(feature = "parallel")]
     fn roots_of_unity(&self, root: F) -> Vec<F> {
-        crate::domain::utils::compute_powers(self.size as usize, root)
+        // TODO: Understand why this functions output isn't domain.elements(),
```

Comment:
The corresponding test fails? If so we should add it anyway, but mark it with an `#[ignore]` attribute

Reply:
added!